test/automation-bot.ts

```diff
@@ -119,35 +112,33 @@ describe('AutomationBot', async () => {
         )
 
         it('should fail if called not through delegatecall', async () => {
-            const tx = AutomationBotInstance.addTrigger(1, triggerType, false, 0, triggerData)
+            const tx = AutomationBotInstance.addTriggers(Math.pow(2, 16) - 1, [false], [0], [triggerData])
             await expect(tx).to.be.revertedWith('bot/only-delegate')
         })
 
```

Comment:
add
```
        it('should fail when trying to call emitGroupDetails', async () => {
            const tx = AutomationBotInstance.emitGroupDetails(TriggerGroupType.CONSTANT_MULTIPLE, testCdpId, [])
            await expect(tx).to.be.reverted
        })
```
to explain existance of lock() :)

Reply:
added